rholang/src/test/scala/coop/rchain/rholang/interpreter/RegistrySpec.scala

```diff
@@ -26,6 +28,21 @@ trait RegistryTester extends PersistentStoreTester {
   implicit val errorLog = new ErrorLog()
   implicit val costAccounting =
     CostAccountingAlg.unsafe[Task](CostAccount(Integer.MAX_VALUE))
+
+  def dispatchTableCreator(registry: Registry): RhoDispatchMap =
```

Comment:
This also is a change unrelated to this commit

Reply:
See my previous comments.